### PR TITLE
Improve UX of Console Setting screen 

### PR DESCRIPTION
**Pull request by ThibaudAV**
**Issue**
https://github.com/gravitee-io/issues/issues/6182

**Description**

Create new form-field for toggle similar to mat-form-field
Use it in settings general 
Improve settings general lock icon an tooltips

**Screenshots**
before: 
image

after:
image

before: 
image

after:
image

**Additional context**

image
image

### CODE REVIEW

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/6182-form-field-slide-toggle/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

File: src/shared/components/form-slide-toogle/gio-form-slide-toggle.stories.ts [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+  },
+  appearance: {
+    options: ['standard', 'fill'],
+    control: { type: 'select' },
+  },
+};
+FullWidth.args = {
+  disabled: false,
+  appearance: 'fill',
+};
```
Comment (gaetanmaisse) [inline src/shared/components/form-slide-toogle/gio-form-slide-toggle.stories.ts:214]: same

File: src/shared/components/form-slide-toogle/gio-form-slide-toggle.stories.ts [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+
+SimilarToMatFormField.argTypes = {
+  appearance: {
+    options: ['standard', 'fill'],
+    control: { type: 'select' },
+  },
+};
+SimilarToMatFormField.args = {
+  appearance: 'fill',
+};
```
Comment (gaetanmaisse) [inline src/shared/components/form-slide-toogle/gio-form-slide-toggle.stories.ts:107]: Can be merged with the declaration of `SimilarToMatFormField`

File: src/shared/components/form-slide-toogle/gio-form-slide-toggle.stories.ts [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+SimilarToMatFormFieldWithIcon.argTypes = {
+  appearance: {
+    options: ['standard', 'fill'],
+    control: { type: 'select' },
+  },
+};
+SimilarToMatFormFieldWithIcon.args = {
+  appearance: 'standard',
+};
```
Comment (gaetanmaisse) [inline src/shared/components/form-slide-toogle/gio-form-slide-toggle.stories.ts:140]: same

File: src/shared/components/form-slide-toogle/gio-form-slide-toggle.stories.ts [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+  },
+  appearance: {
+    options: ['standard', 'fill'],
+    control: { type: 'select' },
+  },
+};
+SimilarToMatFormFieldDisabled.args = {
+  disabled: true,
+  appearance: 'fill',
+};
```
Comment (gaetanmaisse) [inline src/shared/components/form-slide-toogle/gio-form-slide-toggle.stories.ts:177]: same

**Review by gaetanmaisse — APPROVED** [review]

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/6182-form-field-slide-toggle/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

**Review by gaetanmaisse — APPROVED** [review]

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/6182-form-field-slide-toggle/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://612657caa8e859003a8a6430-lfcrourlyr.chromatic.com)

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/6182-form-field-slide-toggle/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

**Pull request merged**